src/pagingscrollview/src/NIPagingScrollView.m

```diff
@@ -64,6 +66,7 @@ - (void)commonInit {
   _scrollView = [[UIScrollView alloc] initWithFrame:self.bounds];
   _scrollView.pagingEnabled = YES;
   _scrollView.scrollsToTop = NO;
+  _scrollView.clipsToBounds = NO;
```

Comment:
Yes, we rely on showing the bleed underneath the bounds to show the neighboring images.